vendor/github.com/mongodb/mongo-tools-common/db/bson_stream.go

```diff
@@ -77,6 +79,21 @@ func (dbs *DecodedBSONSource) Next(result interface{}) bool {
 	return true
 }
 
+// NextGBSON unmarshals the next BSON document into result using the official go driver. Returns true if no errors are
+// encountered and false otherwise. This function does NOT zero out the result before writing to it.
+func (dbs *DecodedBSONSource) NextGBSON(result interface{}) bool {
```

Comment:
This is added so we can easily replace it with a generic `Next` once we drop mgo bson support in mongo-tools-common. The slack conversation re: zeroing in `Unmarshal` is related to this function.